### DISCUSSION TITLE
Docsify external md file feature

### DISCUSSION THREAD

**Discussion by viifirm** [docsifyjs/docsify#2405] [Q&A] [answered]
Dear Docsify team, Would you mind indicate me the correct way to render an external md file in the page? such as, if the link is like this `[file](/docs/file.md)` it will render it as md file, but if the link is this: `[file](https://example.com/docs/file.md)`, it will open the hyperlink rather than render the file, how could I make it rendering a file in another server? 

I tried docsify-remote-markdown plugin, but it need to add tag to enable the specific file, if need to add many external file it will be not handy. please let me know if there are some parameters or format can be use in this senario. Thank you (such as the embed-file function, add a parameter after the link.)

**Comment by sy-records** [comment, marked as answer]
see https://docsifyjs.org/#/embed-files

you missing `:include`

```
[content.md](https://gist.githubusercontent.com/anikethsaha/f88893bb563bb7229d6e575db53a8c15/raw/content.md ':include')
```

**Reply by viifirm** [reply]
Yes i reviewed that option, but that will render the md file at the same page, rather than act like a link to another md file. but thanks anyway, i figured out the way to do it. its using the url rewrite function of web server, rewrite a virtual local dir into a https url, that works perfectly.
